### ISSUE TITLE
Replica + PostgresShardGeneratedIDAutoField failse

### ISSUE THREAD

**Issue by M1hacka** [JBKahn/django-sharding#71]
Hello.
Here's a bug:
![image](https://user-images.githubusercontent.com/2315339/33818769-99938e5a-de68-11e7-82ca-a7113493c912.png)
The problem occures, if I add replica to one of the shards, and use `PostgresShardGeneratedIDAutoField`at the same time. The reason is, that replicas have no `SHARD_ID` assigned, and iteration falls in `KeyError`.

**Comment by M1hacka** [comment]
I suppose, it should be `db_settings.get('SHARD_ID')` if replica is ignored, or something based on RountingStartegy, if replica is used.

**Comment by JBKahn** [comment]
Good catch, I'll update it in addition to the others. It's a busy few weeks for me right now but I'll slot in some time soon.

**Comment by M1hacka** [comment]
A year has passed. Any updates on this issue?

**Comment by JBKahn** [comment]
Yeah, I'll get on this. Sorry I've been rather busy and when it wasn't the holiday season it slipped my mind.
